tfjs-backend-wasm/src/cc/backend.cc

```diff
@@ -90,6 +91,11 @@ EMSCRIPTEN_KEEPALIVE
 #endif
 void init() { xnn_initialize(nullptr); }
 
+#ifdef __EMSCRIPTEN__
+EMSCRIPTEN_KEEPALIVE
+#endif
+const size_t get_threads_count() { return backend::threads_count; }
```

Comment:
FYI. There is a pthreadpool API to query the number of threads in a thread pool

https://github.com/Maratyszcza/pthreadpool/blob/master/include/pthreadpool.h#L86

Reply:
done.